addons/l10n_in_edi/models/einvoice_api.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import pytz
+
+from odoo import fields, models, api, _
+from odoo.exceptions import AccessError
+
+from odoo.addons.iap import jsonrpc
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+DEFAULT_IAP_ENDPOINT = "https://l10n-in-edi.api.odoo.com"
+DEFAULT_IAP_TEST_ENDPOINT = "https://l10n-in-edi-demo.api.odoo.com"
+
+
+class L10nInEdiWebService(models.AbstractModel):
+    _name = "l10n.in.edi.web.service"
+    _description = "E-invoice API (IN)"
```

Comment:
I think it will be more elegant to have this either as a basic python class managing the web service, either as 
 methods in account.edi.format but I don't see any benefit to make a separated `AbstractModel` for this. It makes the code quite :spaghetti:

Reply:
okay, let me convert to  basic python class

Reply:
@smetl You must hate the ORM a lot....   Let's not mix 2 different concepts and just use the AbstractModel here please.  @qdp-odoo